src/Components/Artwork/Details.tsx

```diff
@@ -222,7 +222,7 @@ export const Details: React.FC<DetailsProps> = ({
         <Flex flexDirection="row">
           <Join separator={<Spacer x={1} />}>
             {!hideLotLabel && (
-              <Text variant="xs">
+              <Text variant="xs" style={{ whiteSpace: "nowrap" }}>
```

Comment:
`non-blocking:` I *think* `flexShrink={0}` would also be sufficient

Reply:
Oh, nice. Added changes